posthog/models/feedback/survey.py

```diff
@@ -271,6 +272,20 @@ def update_response_sampling_limits(sender, instance, **kwargs):
 def pre_save_survey(sender, instance, *args, **kwargs):
     update_survey_iterations(sender, instance)
     update_response_sampling_limits(sender, instance)
+    ensure_question_ids(instance)
```

Comment:
what if we also do this when loading a survey? so we know that all surveys will have IDs, otherwise, a migration plan?

Reply:
I added a [manual migration file](https://github.com/PostHog/posthog/pull/29209/files#diff-2cf153f6f308718a62e6787a8728a8a6ce0a53a65e8be908bb9177783dd58a38) that will get all surveys and add question IDs to all of them